lib/kuma/macros/see-compat-table.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Inserts a string
+ * warning about unstable specification
+ * @returns {string} Compatibility warning HTML
+ */
+export default function SeeCompatTable() {
+  return `<div class="notecard__experimental">
+    <h4>Експериментальне</h4>
+    <p>
+      <strong>
+        Це <a href='/uk/docs/MDN/Contribute/Guidelines/Conventions_definitions#eksperymentalne'>експериментальна технологія</a>
```

Comment:
```suggestion
        Це <a href="/uk/docs/MDN/Contribute/Guidelines/Conventions_definitions#eksperymentalne">експериментальна технологія</a>.
```
Тоді і тут подвійні?